src/main/scala/com/fulcrumgenomics/util/SampleBarcodeMetric.scala

```diff
@@ -94,7 +94,42 @@ object SampleBarcodeMetric {
 }
 
 /**
-  * Metrics for matching reads to sample barcodes.
+  * Metrics for matching reads to sample barcodes primarily used in [[com.fulcrumgenomics.fastq.DemuxFastqs]].
+  *
+  * NB: the counts of reads assumes that each read corresponds to a single cluster on an Illumina flowcell.  For example,
```

Comment:
Maybe it's time we stopped mimicking Picard's `SampleBarcodeMetric` and just make this make more sense?